packages/create-strapi-starter/utils/build-starter.js

```diff
@@ -4,23 +4,30 @@ const { resolve, join, basename } = require('path');
 const os = require('os');
 const fse = require('fs-extra');
 
-const chalk = require('chalk');
-const generateNewApp = require('strapi-generate-new');
-
 const ora = require('ora');
 const ciEnv = require('ci-info');
+const chalk = require('chalk');
+const PrettyError = require('pretty-error');
 
-const hasYarn = require('./has-yarn');
+const pe = new PrettyError();
 
+const generateNewApp = require('strapi-generate-new');
+
+const hasYarn = require('./has-yarn');
 const { runInstall, runApp, initGit } = require('./child-process');
 const { getRepoInfo, downloadGithubRepo } = require('./fetch-github');
 
 /**
  * @param  {string} filePath Path to starter.json file
  */
 function readStarterJson(filePath) {
-  const data = fse.readFileSync(filePath);
-  return JSON.parse(data);
+  try {
+    const data = fse.readFileSync(filePath);
+    return JSON.parse(data);
+  } catch (err) {
+    console.log(pe.render(err));
```

Comment:
A general comment : you should wrap errors you catch into a readble human error. We don't need to log a nice Error stack but we absolutely need a human readable error first